snapcraft/internal/elf.py

```diff
@@ -35,6 +35,21 @@
 logger = logging.getLogger(__name__)
 
 
+def _ldd_resolve(soname: str, soname_path: str) -> Tuple[str, str]:
+    logger.debug(f"_ldd_resolve: {soname!r} {soname_path!r}")
+
+    # If found, resolve the path components.
+    if soname_path.startswith("/"):
```

Comment:
can you add a comment on why this check is important? Or example output

Reply:
will do